src/Parser/EnhancementShaman/Modules/ShamanCore/Pandemic.js

```diff
@@ -0,0 +1,58 @@
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Module from 'Parser/Core/Module';
+
+const SPELLID = SPELLS.FLAMETONGUE_BUFF.id;
+const PANDEMIC_THRESHOLD = 11500;
+const debug = false;
+
+class Pandemic extends Module {
+  ft_timestamp = [];
+  early_refresh = [];
+  refresh_avearge;
+  static dependencies = {
+    combatants: Combatants,
+  }
+
+  on_byPlayer_applybuff(event) {
+    if(event.ability.guid === SPELLID) {
```

Comment:
Use the spell reference `SPELLS.FLAMETONGUE_BUFF.id` straight up here is preferred to make it easier to see what spell is being checked.